themes/nord-color-theme.json

```diff
@@ -129,20 +128,6 @@
     "merge.incomingHeaderBackground": "#8fbcbb66",
     "merge.incomingContentBackground": "#8fbcbb4d",
     "merge.border": "#3b425200",
-
-    /* `notification.*` keys are legacy support for VS Code versions >1.21.0 */
```

Comment:
Again like the other deprecated keys removed in this PR: For now Nord supports older versions at least until these keys are removed from VS Code itself or a major update is released making these keys unsupported.
Please revert the change back too.